src/cmd/list.go

```diff
@@ -226,7 +227,14 @@ func getImages() ([]toolboxImage, error) {
 			}
 		}
 
-		if isToolboxImage {
+		for _, image := range toolboxImages {
+			if image.ID == i.ID {
+				isDuplicatedImage = true
+				break
+			}
```

Comment:
It will be better to use a map instead of a loop, because it's a more efficient way of checking membership in a collection.  Since `toolboxImages` isn't of constant size and can potentially be very long, this is essentially becoming a O(n^2) algorithm.  :)

Reply:
Done.